daisy/step.go

```diff
@@ -37,6 +38,8 @@ type Step struct {
 	name string
 	w    *Workflow
 
+	//Run error description
+	ErrorDescription string `json:",omitempty"`
```

Comment:
TimeoutErrorDescription, TimeoutErrorText, or just TimeoutDescription/TimeoutText? The comment should be updated to reflect that this is just used for time out and not other errors

Reply:
Renamed to TimeoutDescription. Also, changed the actual description based on Shawnique's suggestion.